### PR TITLE
Clean up codebase and fix bug

### PR DESCRIPTION
**Pull request by jdpatdiscord**
See commit details for more information

### CODE REVIEW

File: launcher/FileSystem.cpp [hunk 1]
```diff
@@ -414,24 +414,29 @@ bool createShortCut(QString location, QString dest, QStringList args, QString na
...
+    fs::path fullDstPath = dstpath.toStdString();
+    for (auto& entry : fs::recursive_directory_iterator(fullSrcPath))
+    {
+        fs::path relativeChild = fs::relative(entry, fullSrcPath);
+        if (entry.is_directory())
+            if (!fs::exists(fullDstPath / relativeChild))
+                fs::create_directory(fullDstPath / relativeChild);
+        if (entry.is_regular_file())
+        {
+            fs::path childDst = fullDstPath / relativeChild;
```
Comment (LennyMcLennington) [inline launcher/FileSystem.cpp:411]: why is this declared here instead of higher up

Comment (slendidev) [reply launcher/FileSystem.cpp:411]: it's declared inside the if's scope, since it's not used outside of that scope, why declare it anywhere else?

**Comment by slendidev** [comment]
You should signoff your commits with -s

File: launcher/ui/setupwizard/JavaWizardPage.cpp [hunk 2]
```diff
@@ -69,6 +69,7 @@ bool JavaWizardPage::validatePage()
         case JavaSettingsWidget::ValidationStatus::AllOK:
         {
             settings->set("JavaPath", m_java_widget->javaPath());
+            return true;
```
Comment (LennyMcLennington) [inline launcher/ui/setupwizard/JavaWizardPage.cpp]: this looks like it should fall through

**Review by LennyMcLennington — CHANGES_REQUESTED** [review]

**Comment by LennyMcLennington** [comment]
LGTM!!!!!!!!!!!!!!!

**Pull request merged**